codecparsers/mpeg2_parser.cpp

```diff
@@ -356,12 +355,83 @@ namespace MPEG2 {
         DEBUG("time_code_minutes  : %x", m_GOPHeader.time_code_minutes);
         DEBUG("time_code_seconds  : %x", m_GOPHeader.time_code_seconds);
         DEBUG("time_code_pictures : %x", m_GOPHeader.time_code_pictures);
+        DEBUG("closed_gop 	  : %x", m_GOPHeader.closed_gop);
 
         bitReaderDeInit();
 
         return true;
     }
 
+    void Parser::readQuantMatrix(bool& loadMatrix, uint8_t matrix[])
+    {
+
+        bitReaderReadFlag(&loadMatrix);
```

Comment:
Better check return value, what if not enough data from bitstream? Bitstream valid check is important and more important to mpeg2, Because mpeg2 usually from  cable network, it may introduce error.